rock_paper_scissors.py

```diff
@@ -97,11 +97,14 @@ def rounds(self):
 
     @rounds.setter
     def rounds(self, value):
-        if value % 2 != 0:
-            self._rounds = value
-        else:
+        try:
+            value = int(value)
+            if value % 2 == 0 or value < 0:
+                raise ValueError
+        except ValueError:
             print('Please enter odd number of rounds.')
-            exit()
```

Comment:
Oh, no doubt- I'm basically wondering what your expected application flow is after `print`ing.  Are you expecting the program to end, reprompt the user, etc?